README.md

```diff
@@ -18,7 +17,7 @@ indispensable help to this project.
 additional voice helper functions and features for DiscordGo.
 
 * See [dca](https://github.com/bwmarrin/dca) for an **experimental** stand alone
-tool that wraps `ffmpeg` to create opus encoded audio appropriate for use with
+tool that wraps `FFmpeg to create opus encoded audio appropriately for use with
```

Comment:
`appropriate` is correct. It's modifying `audio`, not `create`. It's an adjective, not an adverb.

Reply:
> `appropriate` is correct. It's modifying `audio`, not `create`. It's an adjective, not an adverb.
I ran this through my head a couple of times, but you're definitely correct here. Good catch 👍